cmd/juju/common/model.go

```diff
@@ -23,10 +24,10 @@ type ModelInfo struct {
 	Owner          string                      `json:"owner" yaml:"owner"`
 	Cloud          string                      `json:"cloud" yaml:"cloud"`
 	CloudRegion    string                      `json:"region,omitempty" yaml:"region,omitempty"`
-	ProviderType   string                      `json:"type" yaml:"type"`
+	ProviderType   string                      `json:"type,omitempty" yaml:"type,omitempty"`
 	Life           string                      `json:"life" yaml:"life"`
-	Status         ModelStatus                 `json:"status" yaml:"status"`
-	Users          map[string]ModelUserInfo    `json:"users" yaml:"users"`
+	Status         *ModelStatus                `json:"status,omitempty" yaml:"status,omitempty"`
```

Comment:
This is easier if it's a pointer since status can be empty. The only reason it is not a pointer coming from apiserver (on params) is for backward compatibility. This change stands.